src/Api/BrandsApi.php

```diff
@@ -0,0 +1,350 @@
+<?php declare(strict_types = 1);
+
+namespace SandwaveIo\RealtimeRegister\Api;
+
+use Carbon\Carbon;
+use InvalidArgumentException;
```

Comment:
Please use the internal `SandwaveIo\RealtimeRegister\Exceptions\InvalidArgumentException`

Reply:
Fixed: https://github.com/sandwave-io/realtimeregister-php/pull/25/commits/e55422de0baf9dd204b8f94e628a0e42344372d8